operations/template/logs.tf

```diff
@@ -11,14 +11,24 @@ resource "azurerm_log_analytics_query_pack" "application_logs_pack" {
   location            = data.azurerm_resource_group.group.location
 }
 
-resource "azurerm_log_analytics_query_pack_query" "example" {
+resource "azurerm_log_analytics_query_pack_query" "application_logs" {
   display_name = "TI's Raw Application Logs"
   description  = "View all TI's application logs in a structured format"
 
   query_pack_id = azurerm_log_analytics_query_pack.application_logs_pack.id
   categories    = ["applications"]
 
-  body = "AppServiceConsoleLogs | extend JsonResult = parse_json(ResultDescription) | project-away TimeGenerated, Level, ResultDescription, Host, Type, _ResourceId, OperationName, TenantId, SourceSystem | evaluate bag_unpack(JsonResult)"
+  body = "AppServiceConsoleLogs | project JsonResult = parse_json(ResultDescription) | evaluate bag_unpack(JsonResult) | project-reorder ['@timestamp'], level, message"
```

Comment:
Is there a reason we are no longer removing the `TimeGenerated, Level, ResultDescription, Host, Type, _ResourceId, OperationName, TenantId, SourceSystem` items from the logs?  I'm under the impression that those are worthless columns that don't give us any benefit.

Reply:
Yes, because those columns are not included in the results anymore, so there is no need to remove them. I think the way the query was built before using `extend` was including those columns, but I changed that to use `project` instead

Reply:
Oh!  Hardcore!